app/src/main/java/com/example/kuit6_week1/MainActivity.kt

```diff
@@ -19,29 +37,159 @@ class MainActivity : ComponentActivity() {
+                    .height(height = 60.dp)
+                    .clip(shape = RoundedCornerShape(size = 20.dp))
+                    .background(Color.Black)
+            )
+
+            Box(
+                modifier = modifier
+                    .width(width = 60.dp)
+                    .height(height = 60.dp)
+                    .clip(CircleShape)
+                    .background(Color.Black)
+            )
+        }
+
+        SampleRow(modifier = Modifier.padding(top = 20.dp)) {
+            Box(
+                modifier = Modifier
+                    .size(size = 60.dp)
+                    .border(
+                        width = 2.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(size = 20.dp)
+                    )
+            )
+
+            Box(
+                modifier = Modifier
+                    .size(size = 60.dp)
+                    .border(
+                        width = 3.dp,
+                        color = Color.Black,
+                        shape = CircleShape
+                    )
+            )
+        }
+
+        SampleRow (
+            modifier = Modifier.padding(top = 20.dp)
+        ) {
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .background(Color.Red)
+            )
+
+            Box(
+                modifier = modifier
+                    .size(size = 76.dp)
+                    .border(
+                        width = 1.dp,
+                        color = Color.Black
+                    )
+                    .padding(all = 8.dp)
+                    .clip(CircleShape)
+                    .background(Color.Blue)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 20.dp)
+        ) {
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier.size(size = 80.dp)
+            )
+
+            Image(
+                painter = painterResource(id = R.drawable.image),
+                contentDescription = "개 이미지",
+                modifier = Modifier
+                    .size(size = 80.dp)
+                    .clip(CircleShape)
+            )
+        }
+
+        SampleRow(
+            modifier = Modifier.padding(all = 10.dp)
+        ) {
+            Column (
+                modifier = modifier
+                    .size(240.dp,140.dp)
+                    .border(
+                        1.dp,
+                        color = Color.Black,
+                        shape = RoundedCornerShape(size = 10.dp)
+                    )
+            ){
+                Column(
+                    modifier = Modifier.padding(16.dp),
```

Comment:
검은 테두리와의 간격이 10.dp만큼이라 패딩 10.dp로 주셔야 합니다 ~
```suggestion
                    modifier = Modifier.padding(10.dp),
```